common/djangoapps/student/models.py

```diff
@@ -292,7 +292,7 @@ def age(self):
         year_of_birth = self.year_of_birth
         year = datetime.now(UTC).year
         if year_of_birth is not None:
-            return year - year_of_birth
+            return year - year_of_birth - 1
```

Comment:
Seams like this function and the function below can be encapsulated into a utility function within the class. Can you make this dryer?